services/movie_session.py

```diff
@@ -42,3 +42,9 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: str) -> list:
```

Comment:
The `movie_session_id` parameter should be of type `int` instead of `str` to match the expected data type for IDs in Django models.